c2corg_ui/static/js/map/map.js

```diff
@@ -114,9 +143,19 @@ app.MapController = function($scope, mapFeatureCollection) {
       new ol.layer.Tile({
         source: new ol.source.OSM()
       })
-    ]
+    ],
+    view: new ol.View({
+      center: ol.extent.getCenter(app.MapController.DEFAULT_EXTENT),
+      zoom: app.MapController.DEFAULT_ZOOM
+    })
   });
 
+  /**
+   * @type {ol.View}
+   * @private
+   */
+  this.view_ = this.map.getView();
+
   // editing mode
   if (this['edit']) {
```

Comment:
It seemed related to lines such as
```
this.edit |= false;
```
I have remove them (we don't really need them, if the matching directive attribute is not set, the property is ``undefined`` which evaluates the same way then ``false``) and the error disappeared.